modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -133,18 +149,47 @@ export default class MeshLayer extends Layer {
         accessor: 'getPosition',
         update: this.calculateInstancePositions64xyLow
       },
-      instanceRotations: {
-        size: 3,
-        accessor: ['getYaw', 'getPitch', 'getRoll'],
-        update: this.calculateInstanceRotations
-      },
       instanceColors: {
         size: 4,
         accessor: 'getColor',
         defaultValue: [0, 0, 0, 255]
+      },
+      instanceModelMatCol1: {
+        size: 4,
+        stride: 64,
+        offset: 0,
+        divisor: 1,
+        defaultValue: [1, 0, 0, 0],
+        noAlloc: true
+      },
+      instanceModelMatCol2: {
+        size: 4,
+        stride: 64,
+        offset: 16,
+        divisor: 1,
+        defaultValue: [0, 1, 0, 0],
+        noAlloc: true
```

Comment:
Another AttributeManager API idea: maybe we could add an updater to one of these and the others could reference the first?

```
  instanceModelMatCol1: {
        size: 4,
        stride: 64,
        offset: 0,
        divisor: 1,
        defaultValue: [1, 0, 0, 0],
        update: this.calculateInstanceModelMatrices
      },
      instanceModelMatCol2: {
        size: 4,
        stride: 64,
        offset: 16,
        divisor: 1,
        defaultValue: [0, 1, 0, 0],
        update: 'instanceModelMatCol1'

```